detection-rules/attachment_adobe_image_lure_fts.yml

```diff
@@ -4,14 +4,13 @@ type: "rule"
 severity: "medium"
 source: |
   type.inbound
-  and length(filter(attachments, .file_type not in~ ('png', 'jpeg', 'jpg', 'bmp'))) == 0
-  and length(body.links) >0
+  and all(attachments, .file_type in~ ('png', 'jpeg', 'jpg', 'bmp'))
```

Comment:
These are logically identical but the other one seemed more confusing for me to understand